docs/self-managed/setup/deploy/local/docker-compose.md

```diff
@@ -1,26 +1,254 @@
 ---
 id: docker-compose
 title: "Docker Compose"
-keywords: ["camunda docker"]
+keywords: ["camunda docker", "docker compose"]
 ---
 
-A Docker Compose configuration to run Zeebe, Operate, Tasklist, Optimize, Identity, and Connectors Bundle is available in the [camunda-platform](https://github.com/camunda/camunda-platform/blob/main/docker-compose.yaml) repository.
-Follow the instructions in the [README](https://github.com/camunda/camunda-platform#using-docker-compose).
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
 
-:::warning
-While the [Docker images](/self-managed/setup/deploy/other/docker.md) themselves are supported for production usage, the Docker Compose files are designed to be used by developers to run an environment locally; they are not designed to be used in production. We recommend to use [Kubernetes](/self-managed/setup/install.md) in production.
+A Docker Compose configuration to run Camunda Self-Managed components (Zeebe, Operate, Tasklist, Optimize, Identity, and Connectors).
+
+:::danger
+While the [Docker images](/self-managed/setup/deploy/other/docker.md) themselves are supported for production usage, the Docker Compose files are designed to be used by developers to run an environment locally, and are not designed to be used in production. We recommend [Kubernetes](/self-managed/setup/install.md) for production use cases.
+:::
+
+## Prerequisites
+
+- **Docker Compose:** Version 1.27.0+ (supports the [latest compose specification](https://docs.docker.com/compose/compose-file/)).
+- **Docker:** Version 20.10.16+
+- **Keycloak:** (Local development only). Add Keycloak to resolve to 127.0.0.1 on your local machine, and set `KEYCLOAK_HOST=keycloak` in the `.env` file for token refresh and logout functionality.
+
+## Run Camunda 8 with Docker Compose
+
+To start a complete instance of Camunda 8 Self-Managed environment locally:
+
+1. Clone the [Camunda Self-Managed repository](https://github.com/camunda/camunda-self-managed), and navigate to the directory of the desired Camunda version (for example, `/version/camunda-8.6`).
+2. Run the following command:
+
+```shell
+docker compose up -d
+```
+
+1. Wait for the environment to initialize. This may take several minutes. Monitor the logs, especially the Keycloak container log, to ensure the components have started.
+
+### Configuration options
+
+Running `docker compose up -d` starts all Camunda components, including Identity. The [Camunda Self-Managed repository](https://github.com/camunda/camunda-self-managed) also contains the additional configuration files for lightweight development.
+
+- **docker-compose.yaml:** Contains all Camunda 8 Components for a full stack deployment: Zeebe, Operate, Tasklist, Connectors, Optimize, Identity, Elasticsearch, Keycloak, Web Modeler, and PostgreSQL.
+- **docker-compose-core.yaml:** Contains Camunda 8 Orchestration cluster components: Zeebe, Tasklist, Operate, and Connectors.
```

Comment:
I'm not sure if this file is meant to include what's actually defined as the "Orchestration core" or not, but it appears to only include Zeebe, Operate, Tasklist, Connectors.

Reply:
It's an old name. I will rename it when I work on the changes for Docker Compose 8.7 (which will be done in another PR after this is merged, as it's another task).